get-jira-keys/src/index.ts

```diff
@@ -1,27 +1,47 @@
 import * as github from '@actions/github'
 import * as core from '@actions/core'
 
-const regex = /((([A-Z]+)|([0-9]+))+-\d+)/g;
+
+const githubToken = process.env['GITHUB_TOKEN'];
+const ref = github.context.ref;
+const payload = github.context.payload;
+
+const keyRgx = /((([A-Z]+)|([0-9]+))+-\d+)/g;
```

Comment:
![image](https://user-images.githubusercontent.com/47894313/135043780-2c67ede6-f9f0-43d4-aaf4-bd049bef03ed.png)
No reason for declaring variables in the global scope. It decreases readability. 

**For example:**
Move `keyRgx` to the `mathchKeys` function and change the name of `keyRgx` for something more objective (`jiraKeyRegex`).

Reply:
fixed